src/containers/DiskSpace/DiskSpaceContainer.tsx

```diff
@@ -55,7 +56,11 @@ export const DiskSpace: React.FC = (): ReactElement => {
     }, [selectedInterval, selectedNodeId, nodeList, dispatch]);
 
     if (diskInformation.isLoading) {
-        return <div>Loading</div>;
+        return (
+            <div className="loading-container">
```

Comment:
Not to repeat the code, you could create component `<Loading />` with those divs